### PR TITLE
Adds .NET Standard support to ETW input

### LINKED ISSUE #242: ETW Input Should Support .NET Standard
## Overview
The ETW input provider (Microsoft.Diagnostics.EventFlow.Inputs.Etw) should support .NET Standard as all of its dependencies also support .NET standard.

## Proposed Solution
Change [line 7](../blob/3f045399b0cffc63797a9bbbd9fa0c8e7d15637a/src/Microsoft.Diagnostics.EventFlow.Inputs.Etw/Microsoft.Diagnostics.EventFlow.Inputs.Etw.csproj#L7) to:

```xml
<TargetFrameworks>net451;netstandard1.6;netstandard2.0</TargetFrameworks>
```

### PR DESCRIPTION
**Pull request by commonsensesoftware**
Adds .NET Standard support to ETW input. Resolves #242.

### CODE REVIEW

File: src/Microsoft.Diagnostics.EventFlow.Inputs.Etw/Microsoft.Diagnostics.EventFlow.Inputs.Etw.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <Description>Provides an input implementation for capturing diagnostics data from Event Tracing for Windows (ETW) providers.</Description>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <Authors>Microsoft</Authors>
-    <TargetFramework>net451</TargetFramework>
+    <TargetFrameworks>net451;netstandard1.6;netstandard2.0</TargetFrameworks>
     <AssemblyName>Microsoft.Diagnostics.EventFlow.Inputs.Etw</AssemblyName>
     <VersionPrefix>1.3.3</VersionPrefix>
```
Comment (karolz-ms) [inline src/Microsoft.Diagnostics.EventFlow.Inputs.Etw/Microsoft.Diagnostics.EventFlow.Inputs.Etw.csproj]: Please update the VersionPrefix to 1.4.0

**Comment by karolz-ms** [comment]
You want to ensure that [ETW input tests](https://github.com/Azure/diagnostics-eventflow/blob/master/test/Microsoft.Diagnostics.EventFlow.Inputs.Tests/EtwInputTests.cs) pass on .NET Core 1.1 (netstandard1.6) and .NET Core 2.0 (netstandard2.0). 

Also these tests can probably run only on Windows platform, which has ETW, and not on MacOS or Linux, which do not.

File: src/Microsoft.Diagnostics.EventFlow.Inputs.Etw/Microsoft.Diagnostics.EventFlow.Inputs.Etw.csproj (hunk 1 above)
Comment (commonsensesoftware) [reply src/Microsoft.Diagnostics.EventFlow.Inputs.Etw/Microsoft.Diagnostics.EventFlow.Inputs.Etw.csproj]: Sure

**Comment by commonsensesoftware** [comment]
I ran the tests per the guidelines. .NET Core 2.0 definitely works. I don't have .NET Core 1.1 installed because I don't do work on it anymore. It doesn't look like you have a CI build setup so I guess I'll have to go install it to manually run tests locally. Stand by...

**Comment by karolz-ms** [comment]
I had to do some other work and just took care of this too, see https://github.com/Azure/diagnostics-eventflow/pull/247 

But thanks for inspiration and testing the change! Gives me much more confidence, much appreciated

**Pull request closed without merging**